doc/userguide/upgrade.rst

```diff
@@ -0,0 +1,55 @@
+Upgrading
+=========
+
+General instructions
+--------------------
+
+Suricata can be upgraded by simply installing the new version to the same
+locations as the already installed version. When installing from source,
+this means passing the same ``--prefix``, ``--sysconfdir``,
+``--localstatedir`` and ``--datadir`` options to ``configure``.
+
+::
+
+    $ suricata --build-info|grep -A 3 '\-\-prefix'
+        --prefix                                 /usr
+        --sysconfdir                             /etc
+        --localstatedir                          /var
+        --datarootdir                            /usr/share
+
+
+Configuration Updates
+~~~~~~~~~~~~~~~~~~~~~
+
+New versions of Suricata will occationally include updated config files:
+``classification.config`` and ``reference.config``. Since the Suricata
+installation will not overwrite these if they exist, they should be manually
+updated. If there are no local modifications they can simply be overwritten
+by the ones Suricata supplies.
+
```

Comment:
It's also relevant to some parts of the suricata.yaml as well
Should we also add a diff which settings changed there or just recommend people to use diff and read the changelog/new config?

Reply:
The diff is quite large and a bit too technical I think. I would prefer a more textual overview of changes.